build.xml

```diff
@@ -32,6 +33,19 @@
 		</exec>
 	</target>
 
+	<target name="composer-normalize-check">
```

Comment:
Would you also like me to add a `composer-normalize-fix` target, similar to `cs` and `cs-fix`?